xbmc/pvr/addons/PVRClient.h

```diff
@@ -558,6 +559,23 @@ namespace PVR
      */
     time_t GetBufferTimeEnd() const;
 
+    /*!
+     * @return True if this add-on can be auto-configured via avahi, false otherwise
+     */
+    bool CanAutoconfigure(void) const;
```

Comment:
nope, it's either `CanBeAutoconfigured()` or `CanAutoconfigure()`, so this is correct